core/spec.md

```diff
@@ -5546,6 +5590,43 @@ as defined below:
   - MUST NOT be present if the Resource's `hasdocument` model attribute is
     set to `false.
 
+##### `ancestor` Attribute
+- Type: String
+- Description: The `versionid` of this Version's ancestor.
+
+  The `ancestor` attribute MUST be set to the `versionid` of this Version's
+  ancestor (parent). If this Version is a root of an ancestor hierarchy tree
+  (i.e. it has no parent) then it MUST be set to its own `versionid` value.
```

Comment:
I adjusted this based on your feedback, but now I'm realizing this is the only place where we mention the place parent. Should we do that? Isn't ancestor clear enough?

I suggest omitting (parent) and (i.e. it has no parent).

Reply:
yeah, that's fine